packages/blocks/src/components/drag-handle.ts

```diff
@@ -494,36 +494,54 @@ export class DragHandle extends WithDisposable(LitElement) {
     );
   }
 
-  static calcTarget(
+  /**
+   * Calculates the drop target.
+   */
+  static calcDropTarget(
```

Comment:
I hope this method can be split into smaller pieces, current implementation is a bit too heavy.

Reply:
Ok, I will refactor it.